packages/travix-ui-kit/components/slidingPanel/slidingPanelHeader/slidingPanelHeader.js

```diff
@@ -89,19 +117,27 @@ SlidingPanelHeader.propTypes = {
   dataAttrs: PropTypes.object,
 
   /**
-   * When defined, this custom node appears on the left part of the header
+   * When defined, this custom component will appears on the left part of the header
+   *
+   * Props passed to component
+   * @param className {String}
+   * @param onClick {Function}
    */
-  leftBlock: PropTypes.node,
+  renderLeftBlock: PropTypes.func,
 
   /**
    * Callback for back button
    */
   onBackButtonClick: PropTypes.func,
 
   /**
-   * When defined, this custom node appears on the right part of the header
+   * When defined, this custom component will appears on the right part of the header
+   *
+   * Props passed to component
+   * @param className {String}
+   * @param 'data-rel' {String}
    */
-  rightBlock: PropTypes.node,
+  renderRightBlock: PropTypes.func,
```

Comment:
Forgot to mention, but it would be nice to keep supporting `leftBlock` and `rightBlock` with deprecation warnings suggesting `renderLeftBlock` and `renderRightBlock` instead, so we can keep it backwards compatible for now.

Reply:
Fixed